composer.json

```diff
@@ -115,7 +115,8 @@
                 "Breaks on fresh install": "patches/entitygroupfield-install.patch",
                 "\"Drupal\\group\\Entity\\GroupContent\" unexpected error": "https://www.drupal.org/files/issues/2022-10-14/issue-3315323-3.patch",
                 "Support for Groups v3": "patches/entitygroupfield-groupv3-2.patch",
-                "Broken dropbutton in Groups field when used with Claro": "https://www.drupal.org/files/issues/2022-04-04/entitygroupfield-matching-expected-dropbutton-markup-3183134-6.patch"
+                "Broken dropbutton in Groups field when used with Claro": "https://www.drupal.org/files/issues/2022-04-04/entitygroupfield-matching-expected-dropbutton-markup-3183134-6.patch",
+                "Alter permissions check to use different wording": "patches/entitygroupfield-permissions.patch"
```

Comment:
@nbrettell - I think you forgot to commit the patch file?

Reply:
Good spot! Added